src/homedelivery/query.js

```diff
@@ -52,21 +52,33 @@ async function queryZuora (deliveryDate, config: Config) {
      AND
      (RatePlanCharge.MRR != 0 OR ProductRatePlan.FrontendId__c != 'EchoLegacy')`
     } // NB to avoid case where subscription gets auto renewed after fulfilment time
+
+  /* FIXME: After migration remove the second OR disjunct in the where clause */
```

Comment:
the second OR disjunct can be removed only once the 'holiday end date' of the last old style holiday stop has passed (rather than being related to backfill - since that's just SF - they'll go in as actioned ☑️ )